### ISSUE TITLE
what's the valid mask

### ISSUE THREAD

**Issue by ooochen-30** [Janspiry/Palette-Image-to-Image-Diffusion-Models#79]
@Janspiry i want to train a inpainting diffuison by my own mask, some of them can get rational result but more of them get a black image. i wonder what's the valid mask, can someone give me a help or have somebody miss the same problem

**Comment by Liiiii2101** [comment]
Hey, have you figured out the solution? I'm working on similar tasks

**Comment by ooochen-30** [comment]
i solve my problem by changing the picture storage format from JPEG to PNG. As the jpg format does not guarantee the binarization of images.

**Comment by Liiiii2101** [comment]
Thanks! will give it a try. Can I ask how's the model performance for you?

**Comment by ooochen-30** [comment]
i load the pretrained model of inpainting_places and use 7w+ data, only after serveral epochs it perform great

**Comment by minghuix98** [comment]
So, I think we need to define a new function in 'mask.py' to load our own masks, right? 
And I'm wondering if this generative way can maintain the diversity (that possessed by diffusion models) or not.
Hoping to get your insights about that.
